packages/ariakit/src/tooltip/__examples__/tooltip/test.tsx

```diff
@@ -0,0 +1,20 @@
+import { getByRole, hover, render } from "ariakit-test-utils";
+import Example from ".";
+
+test("render tooltip", () => {
+  render(<Example />);
+  expect(getByRole("button")).toMatchInlineSnapshot(`
+    <button
+      aria-labelledby="r:0"
+      tabindex="0"
+    >
+      Hover on me!
+    </button>
+  `);
+});
+
+test("hover tooltip", async () => {
+  render(<Example />);
+  await hover(getByRole("button"));
+  expect(getByRole("tooltip")).toBeVisible();
+});
```

Comment:
We can also test if the tooltip is shown on focus (need to import `blur` and `press` from `ariakit-test-utils`):
```suggestion
});

test("show tooltip on focus", async () => {
  render(<Example />);
  expect(getByRole("tooltip", { hidden: true })).not.toBeVisible();
  await press.Tab();
  expect(getByRole("tooltip")).toBeVisible();
  await blur();
  expect(getByRole("tooltip", { hidden: true })).not.toBeVisible();
});
```